src/plugin-hooks/useGlobalFilter.js

```diff
@@ -75,9 +75,12 @@ function useInstance(instance) {
     'useExpanded',
   ])
 
-  const setGlobalFilter = filterValue => {
```

Comment:
Change this to `React.useCallback` and you've got yerself a deal 😉

Reply:
Done. I force-pushed my changes to my fork because I assumed you didn't want "code review fix" type commits.

Reply:
😄